platform/server/elasticsearch/ElasticsearchConfig.ts

```diff
@@ -4,6 +4,7 @@ import { readFileSync } from 'fs';
 import { noop } from 'lodash';
 
 import { ClusterSchema } from './schema';
+import { Headers } from '../http/Router/headers';
```

Comment:
Maybe `import { Headers, filterHeaders }` and use the `filterHeaders` below? That already has code for normalizing the headers.

Reply:
I suppose... another idea is to house that method in ElasticsearchConfig.ts as I have it now. What's the idea with the method being in its own file? Do we want to reuse that elsewhere?

Reply:
Does `filterHeaders` need to live in `src/core_plugins/elasticsearch/lib`? Hm. Maybe so.